inference-engine/tests/functional/inference_engine/ir_serialization/serialize.cpp

```diff
@@ -53,7 +53,7 @@ TEST_P(SerializationTest, CompareFunctions) {
 
     bool success;
     std::string message;
-    std::tie(success, message) = compare_functions(result.getFunction(), expected.getFunction(), true, false, true);
+    std::tie(success, message) = compare_functions(result.getFunction(), expected.getFunction(), true, false, true, true, true);
```

Comment:
Could you also add some negative tests cases (probably in other PR). 
I want to see that compare functions doesn't miss any type of attributes. Especially I'm interested in Constants values.

Reply:
This be covered by ticket 47842.